policy/pa.go

```diff
@@ -98,8 +98,13 @@ const (
 	// octets: https://tools.ietf.org/html/rfc1035#page-10. Since two of those octets
 	// are taken up by the leading length byte and the trailing root period the actual
 	// max length becomes 253.
+	// HACK: Right now our schema for the authz table only allows 255 characters
```

Comment:
Can you open a follow-up issue to reference here with a TODO for the proper fix?

Reply:
:+1:

Reply:
I've added a TODO for #3237 because it's most applicable, and we'll just leave that one open and move it out of the sprint rather than closing it.